php/Terminus/Workflow.php

```diff
@@ -123,4 +122,25 @@ public function status($key=null) {
      return $this->status;
    }
 
+   public function isFinished() {
+     if ($this->status->result) {
+       return true;
+     } else {
+       return false;
+     }
+   }
+
+   public function isSuccessful() {
+     if ($this->isFinished() && $this->status->result == 'succeeded') {
```

Comment:
Wait, this is redundant. You're checking to see if this is true and if it == 'succeeded'. if it == "succeeded", php will always cast to bool and = true.